modules/pam_pwhistory/opasswd.c

```diff
@@ -129,6 +129,7 @@ compare_password(const char *newpass, const char *oldpass)
 #ifdef HAVE_CRYPT_R
   struct crypt_data output;
 
+  memset(&output, 0, sizeof(struct crypt_data));
```

Comment:
Also `struct crypt_data output = { 0 }` instead simplifies this statement.

Reply:
Applied.